### ISSUE TITLE
Tracker is not syncing. Gets stuck on fetching dashboard.

### ISSUE THREAD

**Issue by TTNebula** [brandonhellman/mturk-suite#155]
I click sync and it says fetching dashboard but nothing else happens. When I work it doesn't update. Sometimes clicking sync worked. This time it stays stuck of fetching dashboard.

I'm using the latest version of chrome on Windows 10 home.

**Comment by brandonhellman** [comment]
Can you double check to see if you are logged in and then try syncing again?
Is it the Today or 45 Day sync that is getting stuck?

**Comment by TTNebula** [comment]
Both Today and 45 Day Sync are stuck. This started last night when I
noticed the earnings total on the upper left corner of my MTurk dashboard
hadn't been updated. So I went to tracker to manually sync but it got
stuck, I reloaded MTS Tracker, tried again, still stuck. So  I reloaded my
mturk dashboard, and tried again, still stuck. I gave up on it and figured
maybe you were updating something. I went back to it later and it worked!
For both. But I had to keep manually  updating because the totals weren't
adding up automatically. sometimes it worked, sometimes it didn't. I just
assumed you might have been updating it or something. Then today, no matter
what I tried, it stays stuck on fetching.  I tried turning all scripts
except tampermonkey off but that didn't work.
